ome/figure
# Improve file filter

This improves the filtering of figure files in the File > Open dialog as discussed at https://github.com/ome/figure/issues/146.

To test:
 - Open the File > Open dialog
 - Try filtering the files with multiple words as discussed in https://github.com/ome/figure/issues/146
 - Files should appear in the filtered list if all search words are found in the figure name, regardless of order - splitting words on white-space.
